doxia-site-renderer/src/main/java/org/apache/maven/doxia/siterenderer/DefaultSiteRenderer.java

```diff
@@ -434,6 +435,10 @@ protected Context createToolManagedVelocityContext(SiteRenderingContext siteRend
         Locale locale = siteRenderingContext.getLocale();
         String dateFormat =
                 siteRenderingContext.getDecoration().getPublishDate().getFormat();
+        String timezoneId =
+                siteRenderingContext.getDecoration().getPublishDate().getTimezone();
+        TimeZone timezone =
```

Comment:
nit: timeZone

Reply:
I thought about this, but didn't do it in the model: https://maven.apache.org/pom.html `<timezone>`. I can fix Java code, though, if course.